src/Razor/Microsoft.NET.Sdk.Razor/src/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -13,47 +13,54 @@
     <NoWarn>$(NoWarn);NU5100</NoWarn>
     <!-- It is intentional to not include build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets -->
     <NoWarn>$(NoWarn);NU5129</NoWarn>
+    <!-- Cannot run pubternal check on net46 -->
```

Comment:
Then why isn't this conditional on `$(TargetFramework)`?

Reply:
Is there a need to check for pubternals on net5.0? Given this is an SDK, I'm not sure pubternal checks are needed. Maybe @ajaybhargavb @NTaylorMullen @pranavkm can confirm.

Reply:
I guess in this case, the additional `$(DisablePubternalApiCheck)` setting may be warranted. It doesn't make nearly as much sense below.

Reply:
Sounds good